txpool/txpool.go

```diff
@@ -64,6 +64,44 @@ type signer interface {
 	Sender(tx *types.Transaction) (types.Address, error)
 }
 
+// Gauge for measuring pool capacity in slots
+type slotGauge struct {
+	sync.Mutex
+	current uint64
+	limit   uint64
+}
+
+func (g *slotGauge) increase(slots uint64) {
+	g.Lock()
+	defer g.Unlock()
+
+	g.current += slots
+}
+
+func (g *slotGauge) decrease(slots uint64) {
+	g.Lock()
+	defer g.Unlock()
+
+	g.current -= slots
+}
+
+func (g *slotGauge) height() uint64 {
+	g.lock()
+	defer g.unlock()
+
+	return g.current
+}
```

Comment:
Just curious, do you have any special reasons to rename `current` to `height`? It seems to be getter method.

Reply:
Nothing in particular, maybe this approach is more readable:

````
type slots uint64

type slotGauge struct {
	sync.Mutex
	height slots
	max    slots
}
````

and the rename `height()` to `currentHeight()` ?

Reply:
I don't think it's good to rename or add new meaning unless you modify functionality
But actually I'd like to have someone's opinion @zivkovicmilos @lazartravica

Reply:
Agree with @Kourin1996, I don't think there's a reason to make a func named `height` return `current`.

I suggest you rename both the field and getter function name. `current` just seems ambiguous.

Reply:
[Renamed](https://github.com/0xPolygon/polygon-sdk/pull/213/commits/1cf77747ebc905fe88b13247fcd0357562973f9c)